RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Increase border radius on all buttons

## Proposed changes


We’re going to increase the border-radius on all buttons.

To be done

- Increase border radius to 4 on all buttons

## Issue(s)	


## How to test or reproduce


The views and buttons you need to check:
- Profile View
- Edit View
- Reactions, Reply, 
- UiKit's Button, as: answer the koko, reminder, poll
- `Click to join` a call

## Screenshots

#### Before

![Screen Shot 2022-11-07 at 18 21 13](https://user-images.githubusercontent.com/47038980/200419649-2b54fed8-9146-487c-9a68-e525bbe46bd3.png)

#### After

![Screen Shot 2022-11-07 at 18 21 03](https://user-images.githubusercontent.com/47038980/200419672-bd551310-d589-43cb-8feb-5d6128840bdd.png)

## Types of changes




## Checklist



## Further comments
